tests/window.rs

```diff
@@ -0,0 +1,33 @@
+extern crate sample;
+
+use sample::frame::Frame;
+use sample::signal::ConstHz;
+use sample::window::{Window, Windower, HanningWindow, RectangleWindow, WindowType};
+
+#[test]
+fn test_window_at_phase() {
+    let mut window: Window<f64, ConstHz, [f64; 1], HanningWindow> = Window::new(9);
```

Comment:
Rather than having the user have to give the window type type param, it might be nice to provide helper methods/functions? i.e.

```rust
pub fn hanning(...) -> Window<..., Hanning> { ... }
pub fn rectangle(...) -> Window<..., Rectangle> { ... }
```

You could then construct a `Hanning` window via `window::hanning(...)`, etc.